src/index.ts

```diff
@@ -154,6 +154,16 @@ function mergeDefaultOptions<Record, Cursor, Node, CustomEdge extends Edge<Node>
     encodeCursor: (cursor: Cursor) => (cursor as unknown as { id: string }).id,
     decodeCursor: (cursorString: string) => ({ id: cursorString } as unknown as Cursor),
     recordToEdge: (record: Record) => ({ node: record } as unknown as Omit<CustomEdge, 'cursor'>),
+    findManyParamsWithCursor: (args, cursor) => {
+      // Fetch one additional record to determine if there is a next page
+      const take = args.first ? args.first + 1 : -1 * ((args.last || 0) + 1)
+
+      // Convert `before` into prisma `cursor` & `skip`
+      const skip = cursor ? 1 : undefined
+
+      // Pass through the cursor
+      return { take, skip, cursor }
+    },
     ...pOptions,
```

Comment:
I took the generation of these options, and made them somethign a user could have a custom implementation in theory, but in reality its main use is in tweaking the behavior for injecting different behavior into the main function

Reply:
Yep, that looks good. Tiny nit:

```diff
- findManyParamsWithCursor
+ findManyArgumentsWithCursor
```